client/src/app/main/main-home/main-home-page-portal.js

```diff
@@ -15,22 +17,29 @@ class MainHomePagePortal extends React.Component {
 
     render() {
         return (
-            <Widget className={classNames('main-home-page-portal', this.props.className)}>
-                <div className="main-home-page-portal__title">
-                    <Header title={this.props.title || i18n('SUPPORT_CENTER')} description={i18n('SUPPORT_CENTER_DESCRIPTION')} />
-                </div>
-                <div className="main-home-page-portal__cards">
-                    <div className="main-home-page-portal__card col-md-4">
-                        <Card {...this.getTicketsCardProps()}/>
+            <div className="">
+                {queryString.parse(window.location.search)["message"] ?
```

Comment:
can `queryString.parse` return null or undefined? can you check it before accessing `["message"]`?